config/resources/domain.json

```diff
@@ -806,6 +767,68 @@
       },
       "new-resource-base": "http://data.lblod.info/id/vestigingen/"
     },
+    "memberships": {
+      "class": "org:Membership",
+      "relationships": {
+        "member": {
+          "predicate": "org:member",
+          "target": "organizations",
+          "cardinality": "one"
+        },
+        "organization": {
+          "predicate": "org:organization",
+          "target": "organizations",
+          "cardinality": "one"
+        },
+        "role": {
+          "predicate": "org:role",
+          "target": "membership-roles",
+          "cardinality": "one"
+        },
+        "during": {
+          "predicate": "euro:memberDuring",
+          "target": "period-of-time",
+          "cardinality": "one"
+        }
+      },
+      "new-resource-base": "http://data.lblod.info/id/lidmaatschap/"
+    },
+    "membership-roles": {
+      "class": "org:Role",
+      "attributes": {
+        "label": {
+          "predicate": "skos:prefLabel",
+          "type": "string"
+        }
+      },
+      "relationships": {
+        "top-concept-of": {
+          "predicate": "skos:topConceptOf",
+          "target": "concepts",
+          "cardinality": "one"
+        },
+        "has-broader-role": {
+          "predicate": "skos:broader",
+          "target": "membership-roles",
+          "cardinality": "one"
+        }
```

Comment:
**Question:** @bdevloed Do you know why `skos:broader` is used here?

Reply:
To build a kind of hierarchy. lid is the most general membership. And then there are some more special kinds of memberships.